readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -120,6 +120,14 @@ context = {
     'commit': {% if project.repo_type == 'git' %}'{{ commit|slice:"8" }}'{% else %}'{{ commit }}'{% endif %},
 }
 
+# For sphinx >=1.8 we can use html_baseurl to set the canonical URL.
```

Comment:
We should probably update the docs on this as well.

Reply:
Do you mean this https://github.com/readthedocs/readthedocs.org/blob/master/docs/guides/canonical.rst? I didn't find anything else with canonical domain

Reply:
I have updated that guide, let me know if that's ok